set_bucket_tags/requirements.txt

```diff
@@ -0,0 +1,2 @@
+crhelper
+requests
```

Comment:
isn't request a python built in library?

Reply:
No, `requests` is not a built-in python library. It is only its ubiquity that makes it seem that way.